bennu-core/src/main/java/org/fenixedu/bennu/core/domain/UserProfile.java

```diff
@@ -197,6 +198,20 @@ public String getAvatarUrl() {
         return Avatar.mysteryManUrl(this.getUser());
     }
 
+    public String getFullAvatarUrl() {
+        String avatarUrl = getAvatarUrl();
+        if (avatarUrl == null) {
```

Comment:
@joaofreitas-qubit I would actually enforce here `StringUtils.isEmpty()` because a empty string should already return null here.